src/main/java/net/codejava/MvcController.java

```diff
@@ -67,11 +67,12 @@ public String showLoginForm(Model model) {
    * @param user
    */
   private void updateAccountInfo(User user) {

```

Comment:
Edit the javadoc comment for this method so that it reflects the addition of your rewards points feature.

Reply:
Done